lib/src/room.dart

```diff
@@ -666,6 +673,10 @@ class KurentoRoom extends ChangeNotifier implements Room {
   void _handleConnectionState(int trackId, RTCPeerConnectionState state) {
     // REVIEW: Should the room expose the connection state (e.g. `isAgentStreamConnected`, `isExplorerStreamConnected`)?
     _log.info('connection state changed track_id=$trackId state=$state');
+    if (RTCPeerConnectionState.RTCPeerConnectionStateFailed == state) {
+      SfuConnectionDirection direction = _connectionByTrackId.remove(trackId)!.direction;
```

Comment:
Do we need to dispose the removed connection?

Reply:
good question! I'll call the method just in case.